sim/target-agnostic.mk

```diff
@@ -58,10 +58,10 @@ $(GENERATED_DIR):
 
 # empty recipe to help make understand multiple targets come from single recipe invocation
 # without using the new (4.3) '&:' grouped targets see https://stackoverflow.com/a/41710495
-$(VERILOG) $(HEADER) $(fame_annos): $(VERILOG).intermediate
+$(VERILOG) $(HEADER) $(fame_annos): $(VERILOG).intermediate ;
 
 .SECONDARY: $(VERILOG).intermediate
-$(VERILOG).intermediate: $(FIRRTL_FILE) $(ANNO_FILE) $(RUN_MAIN_DEPS)
+$(VERILOG).intermediate: $(FIRRTL_FILE) $(ANNO_FILE) $(PRE_ELABORATION_TARGETS)
```

Comment:
The other missed rename.

Reply:
it wasn't missed now that I rebased and squashed a bunch of stuff together... :)